app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -7,10 +7,14 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       provider_id: '',
       zone: '',
       hostname: '',
+      default_hostname: '',
+      amqp_hostname: '',
       project: '',
-      api_port: '',
+      default_api_port: '',
+      amqp_api_port: '',
       api_version: '',
       security_protocol: '',
+      amqp_security_protocol: '',
```

Comment:
Agree, which is why I think I changed it in https://github.com/ManageIQ/manageiq/pull/7157